modules/_labs/select/react/lib/SelectMenu.tsx

```diff
@@ -57,11 +96,21 @@ const menuBorderCSS = (error?: ErrorType): CSSObject => {
     // Render the divider using a pseudo-element
     '&:before': {
       backgroundColor: colors.soap400,
+      borderLeft: dividerBorder,
+      borderRight: dividerBorder,
+      boxSizing: 'border-box',
       content: '""',
       display: 'block',
       height: 1,
-      borderLeft: dividerBorder,
-      borderRight: dividerBorder,
+      position: 'absolute',
+      width: '100%',
+
+      '[data-popper-placement^="bottom-start"] &': {
+        top: 0,
+      },
+      '[data-popper-placement^="top-start"] &': {
+        bottom: 0,
```

Comment:
I'm curious about the `^` in this selector. Aren't the values only either `top-start` or `bottom-start`?

Reply:
Extraneous code left over from a copy-and-paste. I'll remove the carets.